org.desmume.DeSmuME.json

```diff
@@ -0,0 +1,68 @@
+{
+    "app-id": "org.desmume.DeSmuME",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "20.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "desmume",
+    "rename-desktop-file": "desmume.desktop",
+    "finish-args": [
+        "--filesystem=xdg-config/desmume:create",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--socket=pulseaudio",
+        "--device=dri"
+    ],
+    "modules": [
+        "shared-modules/SDL/SDL-1.2.15.json",
+        {
+            "name": "pcap",
+            "buildsystem": "autotools",
+            "config-opts": [
+                "--disable-shared",
+                "--disable-dbus"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.tcpdump.org/release/libpcap-1.9.1.tar.gz",
+                    "sha512": "ae0d6b0ad8253e7e059336c0f4ed3850d20d7d2f4dc1d942c2951f99a5443a690f0cc42c6f8fdc4a0ccb19e9e985192ba6f399c4bde2c7076e420f547fddfb08"
+                }
+            ]
+        },
+        {
+            "name": "desmume",
+            "buildsystem": "meson",
+            "subdir": "desmume/src/frontend/posix/",
+            "config-opts": [
+                "-Dfrontend-cli=false"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/TASVideos/desmume",
+                    "commit": "b400093e3ccc130fba862ef2842e86143ac37196"
+                }
+            ]
+        },
+        {
+            "name": "appdata",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -Dm644 -t /app/share/appdata/ org.desmume.DeSmuME.appdata.xml"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "path": "org.desmume.DeSmuME.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html